src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -242,10 +245,17 @@ private boolean areTotalsEqual(InvoiceLine invoiceLine1, InvoiceLine invoiceLine
               OrderInvoiceRelationship orderInvoiceRelationship = new OrderInvoiceRelationship();
               orderInvoiceRelationship.withInvoiceId(invoiceLine.getInvoiceId()).withPurchaseOrderId(poLine.getPurchaseOrderId());
 
-              return orderService.deleteOrderInvoiceRelationshipByInvoiceIdAndLineId(invoiceLine.getInvoiceId(), invoiceLineFromStorage.getPoLineId(), requestContext)
-                        .thenCompose(v -> orderService.createOrderInvoiceRelationship(orderInvoiceRelationship, requestContext)
-                                                      .thenCompose(relationship -> CompletableFuture.completedFuture(null))
-                        );
+              return CompletableFuture.supplyAsync(() -> {
```

Comment:
Here as I know response will be returned immediately and there is no chance to know that relations were deleted or created.
Also please use org.folio.completablefuture.FolioVertxCompletableFuture instead of CompletableFuture, because future will be unbound from Vertx context and it will be a reason of resources leaks.